packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -145,34 +176,41 @@ export class VaTelephone {
   }
 
   render() {
-    const { contact, extension, notClickable, international, tty, vanity } = this;
+    const { contact, extension, notClickable, international, tty, sms, vanity } = this;
     const formattedNumber = VaTelephone.formatPhoneNumber(
       contact,
       extension,
       international,
       vanity,
-      tty
+      tty,
+      sms
     );
     const formattedAriaLabel = `${VaTelephone.formatTelLabel(
       contact,
       extension,
       tty,
-      international
+      international,
+      sms
     )}.`;
 
+    const telLabel = `${VaTelephone.createTelLabel(tty, sms)}`;
+
     return notClickable ? (
       <Fragment>
         <span aria-hidden="true">{formattedNumber}</span>
         <span class="sr-only">{formattedAriaLabel}</span>
       </Fragment>
     ) : (
-      <a
-        href={VaTelephone.createHref(contact, extension)}
-        aria-label={formattedAriaLabel}
-        onClick={this.handleClick.bind(this)}
-      >
-        {formattedNumber}
-      </a>
+      <Fragment>
+        {telLabel ? <span>{telLabel}</span> : ''}
```

Comment:
I looked at the original issue and it was unclear to me if the feedback was just to add "Text" in front of the number like we do for TTY, or if we should also make "TTY" and "Text" _not_ part of the clickable link. Is this specified somewhere? For background, in May [Randi made "TTY" part of the clickable text in the content style guide](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/pull/800).

![The phone numbers section of the content styleguide. The markup for a TTY number is highlighted, with an extra highlight showing that the "TTY" preceding text is inside of the clickable link](https://user-images.githubusercontent.com/2008881/198372565-7537a94b-91b8-4909-9974-501b434566b0.png)

Reply:
@bkjohnson You can review the discussion about this update in the [Slack thread here](https://dsva.slack.com/archives/G01BJ3ESXL4/p1666708186077849?thread_ts=1666707265.753579&cid=G01BJ3ESXL4) and this was added to the [original ticket as a comment here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1246#issuecomment-1290671223). If a follow-up ticket needs to be created to update the documentation about this, then we should let the team know.

Reply:
> Here is the way we should format SMS:
> 
> Text: [838255](url)
> 
> And we should update TTY to be consistent with the phone number format and SMS format:
> 
> TTY: [711](url)

I read the comment, but I'm saying that without a clear statement like "the type of link needs to be its own element so that it isn't clickable" the hyperlink formatting there could be an accident.

The slack discussion helps a little, but it seems like it's assuming that regular numbers will have "Call:" in front:

>  and do not link Text nor TTY **to be consistent with Call.**

We don't put "Call:" in front of the numbers though. The reason "TTY:" was added is because it is typically paired with a non-tty number, [like this](https://github.com/department-of-veterans-affairs/vets-website/blob/f62cb89618190a6eda0230092b1b782aa86ba29b/src/applications/appeals/995/content/GetFormHelp.jsx#L8-L11).

![Screenshot of a TTY number appearing in parentheses after a number meant for a hearing or speech-enabled user.](https://user-images.githubusercontent.com/2008881/198405818-2505604d-5be0-4c76-91a3-1b840c41deb2.png)

Reply:
Thanks for that context @bkjohnson . I should have asked for more examples in the Slack thread. I suspect removing "TTY:" from the hyperlink would make that tap target too small and more difficult to hit. Sometimes we say "call TTY: 711" within a sentence and there having a larger target to see does seem useful. So let's leave TTY alone.

Also, "TTY:" is more of a label that just happens to be the same as the protocol type. To make matters more complicated, users can now switch a phone call to [RTT](https://support.google.com/accessibility/android/answer/9042284?hl=en) (Real-time text) which runs over TTY. In which case, I'm not sure which number they are supposed to call to do that.

Anyway, back to "text". I don't think "SMS: [number]" makes sense as not all users will know that protocol means send a text. Thus "text" is more like "call" which we don't use in the hyperlink, don't inject, and does not use a colon. Though I can find instances where we violate the TTY rule and where a colon could be used:

<img width="523" alt="Screen Shot 2022-10-28 at 11 34 17 AM" src="https://user-images.githubusercontent.com/52176351/198676573-07fded36-d5ef-40bd-b55e-9846e966907d.png">

At any rate, apologies for pulling a 180 here but let's go with leaving text out. Folks can add "Text" or "text" ahead of the number just as they do with phone numbers.